src/components/Login/LoginContainer.tsx

```diff
@@ -129,10 +129,12 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
     if (!isLoading && token && endpoint && features?.FLOW_TYPE) {
       switch (features?.FLOW_TYPE) {
         case "DEFAULT":
-          navigation.navigate("CollectCustomerDetailsScreen");
-          break;
         case "MERCHANT":
-          navigation.navigate("MerchantPayoutScreen");
+          // TODO: Navigate directly to CampaignInitialisationScreen on successful login.
```

Comment:
Seems like a significant refactoring. Curious to know -- what is the impact of this workaround for now?

Reply:
Not a lot of impact in terms of functionality. Just that right now the Login component is knowing a lot of things, including details on the features + products. It actually just needs to know if the said user is authenticated and authorised to continue. The retrieval of the campaign config can then be deferred to the CampaignInitialisation component.

By splitting these concerns, in future when users are tagged to multiple campaigns, then we can call the CampaignInitialisation component only when the user selects the campaign. Login will have no part to play in the campaign config.

Reply:
Makes sense! I wonder if it is worth making this a chore rather than leave it as a TODO, because I am wary about TODOs just living in the codebase for years after they are added 🙈  But will leave this as a discussion on Thurs~

Reply:
Yea, I think for this part, I'll surely move it out when we migrate the backend over to the new campaign config. 🤣